test/run

```diff
@@ -11,6 +11,8 @@ if [ -n "$TEST_SCENARIO" ]; then
     test/check-cli TestQcow2
   elif [ "$TEST_SCENARIO" == "tar" ]; then
     test/check-cli TestTar
+  elif [ "$TEST_SCENARIO" == "repos" ]; then
+    test/check-cli TestRepos
```

Comment:
This should go after line 20, similar to check-api. VM setup is a heavy operation and if we can squeeze the tests in the available window without them timing out then this is what we want to be doing as the first step.

Otherwise we also need to update `testmap.py` which makes the whole process a bit more cumbersome.